examples/datadog-express-apm/host-agent/env.sample

```diff
@@ -4,7 +4,8 @@ CTP_CLIENT_ID=
 CTP_CLIENT_SECRET=
 CTP_AUTH_URL=https://auth.<your-region>.gcp.commercetools.com
 CTP_API_URL=https://api.<your-region>.gcp.commercetools.com
-
+CTP_CLIENT_USERNAME=<customer_username>
+CTP_CLIENT_PASSWORD=<customer_password>
```

Comment:
I believe these environment entries are not needed here. Can we remove them to avoid confusion?